app/views/editor/rich.html.erb

```diff
@@ -186,6 +235,13 @@
 </div>
 <script>
 
+  $( document ).ready(function() {
+      $("#map_content").hide() ;
+      $("#location_button").click(function(){
+          $("#map_content").toggle() ;
+      }) ;
+  });
+
```

Comment:
Down on line ~257, could we insert some params from the tags if they exist? Like, if there are already `lat:___` and `lon:____` tags in format `/post?tags=one,two,lat:90,lon:-90` we could detect them and insert them here? We can just supply them in format:

```js
editor = new PL.Editor({
  lat: 90,
  lon: -90,
  ...
```

for now, and develop the detection and response code for it later in the Editor itself. How does that sound?

Reply:
Awesome ! I guess we are trying to solve the case when we edit a document , right ?
Currently default location on map will be shown and not the existing one .

Reply:
I think we might want to do either edit or create new -- both will open with this template. So if we add this option, we'll be able to insert either pre-populated ones from a link, or the default ones if it's editing an existing node. How does that sound?

Reply:
Makes sense 😄 !